### ISSUE TITLE
Spontaneously disconnected conveyers

### ISSUE THREAD

**Issue by wolfrasin** [MaFi-Games/Captain-of-Industry-issues#1190]
### Issue description

I keep finding my setups broken by conveyers that no longer convey, no idea why. I backtrack from the empty bin and find a conveyer that was working before, feeding a machine, now stopped and full "not connected." Upgrading does nothing. Deleting and rebuilding the connection point does nothing.  I keep having to delete the whole segment and reconnect it from the spine to the machine to make it flow again. This is effecting the u-shaped conveyers, haven't noticed a broken flat belt. 

This is new since 5.3. I'm having significantly less fun with this bug in play

### Reproduction steps

[broken concreteeee.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/11928594/broken.concreteeee.zip)

2. Find my concrete factory and check out the broken u-belts connecting the grinders to the mixers.
3. Not sure how long those have been down, they just worked until I caught the concrete slab box emptying. The one I had to fix before this was the u-belt moving Animal Feed to the Mixer II producing Compost by the Chicken Farms. Before that it was the u-belt feeding into the Copper Ore Grinder  (they're working at this save) 
4. The Animal feed u-belt was recently paused when it broke. The Copper Ore Grinder feed belt full before breaking when the Ground Copper belt built and the factory restarted.

### Game version and build number

v0.5.3c (b222)

### Operating system

Windows

**Comment by MaFi-Games** [comment]
Thanks for the report, unfortunately I was unable to see the broken belts in your save, is it possible that reloading the game fixes it? Some more info about this issue would be appreciated.

Is this the location of the issue?
![image](https://github.com/MaFi-Games/Captain-of-Industry-issues/assets/90082857/a4b06227-e04a-49f9-849b-d454e33a0ace)

**Comment by wolfrasin** [comment]
Yes, that's the spot. When you opened up the save, the gravel, sand and crushed slag were moving on the belts, connected? 
I just opened up the save again and found the belts working too.
The issue must have something to do with how long the game's been running. I will add more if I get more if it happens again and I get more information
